Jenkinsfile

```diff
@@ -3,45 +3,106 @@
 
 standardProperties()
 
-def runTests(String name, String bedrockCommand, Boolean runAll) {
-  // Clean out the old XML files before running tests, since we junit import *.XML files
-  dir('scratch') {
-    if (isUnix()) {
-      sh "rm -f *.xml"
-    } else {
-      bat "del *.xml"
+def withRemoteCreds(String provider, Closure body) {
+  if (provider == 'aws') {
+    withAwsCreds {
+      body()
+    }
+  }
+  if (provider == 'lambdatest') {
+    withLtCreds {
+      body()
+    }
+  }
+}
+
+def withAwsCreds(Closure body) {
```

Comment:
Can we stick these in waluigi?

Reply:
Moved to `bedrockRemoteTools.withRemoteCreds(provider)` in waluigi